src/index.ts

```diff
@@ -86,17 +141,17 @@ if (stylesheet) {
  */
 const processFile = (filePath: string, isIndex: boolean): string => {
   const extension = path.extname(filePath).toLowerCase();
-  if (extension !== '.txt') {
+  if (extension !== '.txt' && extension !== '.md') {
     return '';
   }
 
+  const isMd = extension === '.md';
   const text = fs.readFileSync(filePath, 'utf-8');
 
   // title is before the first 2 blank lines of the text
   const titleAndContent = text.split(/\n\n\n/);
```

Comment:
The title should only be parsed like this when the input file is `.txt`. Heading in markdown doesn't have to be before the first two blank lines. I'd say you can skip parsing the title if the file is `.md`.

Reply:
sure